startsnaphelper/build.gradle

```diff
@@ -1,7 +1,8 @@
 apply plugin: 'com.android.library'
-
 apply from: "$rootDir/android.gradle"
 
+group = 'com.github.redspace-android'
```

Comment:
Does this not work if added to `android.gradle`?  Fine if not but worth considering.